mappings/net/minecraft/client/gl/GlShader.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS none/bwn net/minecraft/client/gl/GlShader
+	CLASS none/bwn$a Type
+		FIELD a VERTEX Lnone/bwn$a;
+		FIELD b FRAGMENT Lnone/bwn$a;
+		FIELD c name Ljava/lang/String;
+		FIELD d fileExtension Ljava/lang/String;
+		FIELD e shaderType I
+		FIELD f nameObjectMap Ljava/util/Map;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;Ljava/lang/String;I)V
+			ARG 0 name
+			ARG 1 fileExtension
+			ARG 2 shaderType
+		METHOD a getName ()Ljava/lang/String;
+		METHOD b getFileExtension ()Ljava/lang/String;
+		METHOD c getShaderType ()I
+		METHOD d getNameObjectMap ()Ljava/util/Map;
+	FIELD a shaderType Lnone/bwn$a;
+	FIELD b name Ljava/lang/String;
+	FIELD c shaderRef I
+	FIELD d refCount I
+	METHOD <init> (Lnone/bwn$a;ILjava/lang/String;)V
+		ARG 0 shaderType
+		ARG 1 shaderRef
+		ARG 2 name
+	METHOD a getName ()Ljava/lang/String;
+	METHOD a incRef (Lnone/bwk;)V
```

Comment:
We might have some other places where we use `incr` and `decr`, so perhaps search for them and change for consistency then?